src/Build/BackEnd/Shared/BuildRequestConfiguration.cs

```diff
@@ -557,61 +526,35 @@ public Lookup BaseLookup
             {
                 ErrorUtilities.VerifyThrow(!IsCached, "Configuration is cached, we shouldn't be accessing the lookup.");
 
-                if (null == _baseLookup)
-                {
-                    _baseLookup = new Lookup(Project.ItemsToBuildWith, Project.PropertiesToBuildWith);
-                }
-
-                return _baseLookup;
+                return _baseLookup ??
+                       (_baseLookup = new Lookup(Project.ItemsToBuildWith, Project.PropertiesToBuildWith));
```

Comment:
Does this pattern produce better codegen? I think the old way is more readable.

Reply:
R# likes one-liners (well, 2 in this case) over 4-liners. Reverted.